provisioning/service/src/ProvisioningServiceClient.cs

```diff
@@ -451,15 +452,13 @@ public Task<DeviceRegistrationState> GetDeviceRegistrationStateAsync(string id,
         /// provided <see cref="DeviceRegistrationState"/> information. The Device Provisioning Service will care about the
         /// id and the eTag on the DeviceRegistrationState. If you want to delete the DeviceRegistrationState regardless the
         /// eTag, you can use the <see cref="DeleteDeviceRegistrationStateAsync(string, CancellationToken)"/> passing only the id.
-        ///
-        /// If the id does not exists or the eTag does not matches, this method will throw
-        /// <see cref="ProvisioningServiceClientException"/>.
         /// </remarks>
         /// <param name="deviceRegistrationState">The <see cref="DeviceRegistrationState"/> that identifies the DeviceRegistrationState.
         /// It cannot be <c>null</c>.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <exception cref="ProvisioningServiceClientException">If the Provisioning Device Service was not able to
-        /// delete the registration status information for the provided DeviceRegistrationState.</exception>
+        /// <exception cref="DeviceProvisioningServiceException">
+        /// If the service was not able to delete the registration status information for the provided DeviceRegistrationState.
```

Comment:
Sounds kinda circular and clunky. I'd just say:
```suggestion
        /// When the service wasn't able to delete the registration status.
```